contrib/globalsign/mgo/mgo.go

```diff
@@ -38,24 +38,25 @@ func Dial(url string, opts ...DialOption) (*Session, error) {
 		version = info.Version
 	}
 
-	// Note that these are all currently known hosts that are alive
-	// This is not guaranteed to be the exact server involved in the communication
-	var hostnames []string
-	for _, addr := range session.LiveServers() {
-		host, _, err := net.SplitHostPort(addr)
-		if err == nil {
-			hostnames = append(hostnames, host)
+	tags := map[string]string{
+		"mgo_version": version,
+	}
+	if ls := session.LiveServers(); len(ls) > 0 {
+		tags["hosts"] = strings.Join(ls, ", ")
```

Comment:
Could you double check if the whitespace in `", "` is necessary?

Reply:
Good catch, removed